ironfist/src/cpp/spell/spells.h

```diff
@@ -99,24 +97,21 @@ enum Spell : int
 };
 
 
-enum SPELL_ATTRIBUTES
-{
+enum SPELL_ATTRIBUTES{
   ATTR_COMMON_SPELL = 0x1,
   ATTR_COMBAT_SPELL = 0x2,
   ATTR_ADVENTURE_SPELL = 0x4,
   ATTR_DURATIONED_SPELL = 0x8,
 };
 
 
-enum SPELL_CATEGORY
-{
+enum SPELL_CATEGORY{
   SPELL_CATEGORY_COMBAT = 0,
   SPELL_CATEGORY_ADVENTURE = 1,
   SPELL_CATEGORY_ALL = 2,
 };
 
-enum STACK_MODIFYING_EFFECT
-{
+enum STACK_MODIFYING_EFFECT{

```

Comment:
@zhxsxuan I'd like you to have a look over https://google.github.io/styleguide/cppguide.html#Formatting again in case you missed anything else. There needs to be a space before the open brace.